subsys/net/ip/tcp.c

```diff
@@ -1204,6 +1204,9 @@ bool net_tcp_ack_received(struct net_context *ctx, u32_t ack)
 			}
 		}
 
+		NET_DBG("[%p] ACK received pkt %p (%zd bytes)", ctx->tcp,
```

Comment:
Just reading this message, I'm not sure how to interpret info it provides. "ACK received pkt"? Maybe "Received ACK pkt"? What does "NNN bytes" printed signifies?

Reply:
As the function name implies, is called when we receive an ACK message and here the print means that ACK was received and pkt was %p and it was %zd bytes long.

Reply:
Yeah, the matter is that "ACK" is just a flag in a packet, the only other relevant piece from the pkt is ack_no. All the other info is not related to ACK, hopefully someone reading that log message will remember that very well. (E.g. "%zd bytes" printed there isn't related to how many bytes was acked).